ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Node.hs

```diff
@@ -159,21 +159,26 @@ protocolInfoByron genesisConfig mSigThresh pVer sVer mLeader =
     ProtocolInfo {
         pInfoConfig = TopLevelConfig {
             topLevelConfigProtocol = PBftConfig {
-                pbftParams = byronPBftParams genesisConfig mSigThresh
+                pbftParams = byronPBftParams compactedGenesisConfig mSigThresh
               }
           , topLevelConfigBlock = FullBlockConfig {
                 blockConfigLedger = genesisConfig
```

Comment:
```suggestion
                blockConfigLedger = compactedGenesisConfig
```

This can also be `compactedGenesisConfig` right? Or else we still keep a reference to the balances through the `PartialLedgerConfig`.

Reply:
Indeed! Good catch, thanks